models/index.js

```diff
@@ -1,5 +1,10 @@
 var mongoose = require("mongoose");
 
 // connect mongoose to your Mongo DB here
+DB_CONN="mongodb://mejiamiguel4:cocacola@ds161584.mlab.com:61584/wdi12mejiamiguel4"
```

Comment:
Noooooo!!!!! This should go in a .env file that is in the .gitignore file so it never gets pushed here!!